### PR TITLE
Add dependabot

### PR DESCRIPTION
**Pull request by oriontvv**
This PR adds dependabot config

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: 2
+updates:
+  - package-ecosystem: "cargo"
```
Comment (JohnTitor) [inline .github/dependabot.yml:3]: Could you exclude tempdir?

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 10
+  - package-ecosystem: "docker"
```
Comment (JohnTitor) [inline .github/dependabot.yml:8]: We don't have any Dockerfile here.

File: .github/dependabot.yml (hunk 1 above)
Comment (oriontvv) [reply .github/dependabot.yml:3]: Yes, done.

**Review by JohnTitor — APPROVED** [review]
Thank you!

**Pull request merged**